examples/00-core_examples/00-basic_dpf_implementation.py

```diff
@@ -0,0 +1,217 @@
+#       'fields_container_factory', 'fields_factory', 'generic_data_container', 'generic_support','normalize_vector',
+#       'create_point', 'incremental', 'inputs', 'meshed_region', 'meshes_container', 'mesh_info',
+#       'mesh_scoping_factory', 'model', 'nodes', 'operator_specification', 'outputs', 'path_utilities', 'plotter',
+#       'plugins', 'property_field', 'results', 'result_info', 'runtime_config', 'scoping', 'scopings_container',
+#       'server', 'server_context', 'server_factory', 'server_types', 'session', 'settings', 'streams_container',
+#       'string_field', 'support', 'time_freq_scoping_factory', 'time_freq_support', 'unit_system', 'workflow'
+
+######################################################################################################################
+# # ~~~~~~~ USING DPF ~~~~~~~ # #
+
+# An operator is the main object here. You use them to load, operate on, and output data.
+
+# Each operator contains input and output pins, like those in an integrated circuit in electronics, that
+# submit data to the operator and output the computed result from the operator, with respect to its description.
+# Each pin has a number and a name
+
+# The basic steps for using the operators are:
+# ---------------------------------------------------------------------------------------------------------------------
+# 1) Instance the operator. There are two ways for doing this:
+#
+#     1.1) By instantiating any of the derived classes available in the 'ansys.dpf.core.operators' package.
+#     This package also have its subpackages:
+#             - 'ansys.dpf.core.operators.averaging' package
+#             - 'ansys.dpf.core.operators.compression' namespace
+#             - 'ansys.dpf.core.operators.filter' package
+#             - 'ansys.dpf.core.operators.geo' package
+#             - 'aansys.dpf.core.operators.invariant package' package
+#             - 'ansys.dpf.core.operators.logic' package
+#             - 'ansys.dpf.core.operators.mapping' package
+#             - 'ansys.dpf.core.operators.math' package
+#             - 'ansys.dpf.core.operators.mesh' package
+#             - 'ansys.dpf.core.operators.metadata' package
+#             - 'ansys.dpf.core.operators.min_max' package
+#             - 'ansys.dpf.core.operators.result' package
+#             - 'ansys.dpf.core.operators.scoping' package
+#             - 'ansys.dpf.core.operators.serialization' package
+#             - 'ansys.dpf.core.operators.server' package
+#             - 'ansys.dpf.core.operators.utility' package
+
+#      If you believe you'll be using multiple modules available in the same package, its better to import it all
+#      at the beginning
+        from ansys.dpf.core import operators as ops # Import of the dpf core subpackage 'ansys.dpf.core.operators'
+        my_operator = ops.subpackageAttribute.operatorAttribute() # I use the word attribute for any name following a dot
```

Comment:
So, one thing to keep in mind, is that the examples are run in the CI. So they need to actually be working scripts.
If you want to make a generic example, you can put it as text in a comment.